internal/pkg/flink/internal/store/store_utils.go

```diff
@@ -136,7 +136,7 @@ func parseSetStatement(statement string) (string, string, error) {
 
 	indexOfSet := strings.Index(strings.ToUpper(statement), config.ConfigOpSet)
 	if indexOfSet == -1 {
-		return "", "", &types.StatementError{Msg: "Error: Invalid syntax for SET. Usage example: SET 'key'='value'."}
+		return "", "", &types.StatementError{Msg: "Invalid syntax for SET. Usage example: SET 'key'='value'."}
```

Comment:
I wonder if the backend could parse this for us, even though it's a frontend-specific command.